test/test_data.py

```diff
@@ -5,91 +5,390 @@
+    assert streams[i]["info"]["type"][0] == "data"
+    assert streams[i]["info"]["channel_count"][0] == "1"
+    assert streams[i]["info"]["channel_format"][0] == "int32"
+    assert streams[i]["info"]["source_id"][0] == "test_stream.py:191695:0"
+    assert streams[i]["info"]["nominal_srate"][0] == "1.000000000000000"
+    assert streams[i]["info"]["version"][0] == "1.100000000000000"
+    assert streams[i]["info"]["created_at"][0] == "91690.55328314200"
+    assert streams[i]["info"]["uid"][0] == "bc60b7bb-e632-407c-b3db-e42f9cad4179"
+    assert streams[i]["info"]["session_id"][0] == "default"
+    assert streams[i]["info"]["hostname"][0] == "kassia"
+    assert streams[i]["info"]["v4address"][0] is None
+    assert streams[i]["info"]["v4data_port"][0] == "16573"
+    assert streams[i]["info"]["v4service_port"][0] == "16573"
+    assert streams[i]["info"]["v6address"][0] is None
+    assert streams[i]["info"]["v6data_port"][0] == "0"
+    assert streams[i]["info"]["v6service_port"][0] == "0"
+    assert streams[i]["info"]["desc"][0]["manufacturer"][0] == "pylsltools"
+    channels = streams[i]["info"]["desc"][0]["channels"][0]
+    assert channels["channel"][0]["label"][0] == "ch:00"
+    assert channels["channel"][0]["type"][0] == "misc"
+
+    # Info added by pyxdf
+    assert streams[i]["info"]["stream_id"] == 4
+    assert streams[i]["info"]["effective_srate"] == pytest.approx(1)
+    assert streams[i]["info"]["segments"] == [(0, 9)]
+
+    # Footer
+    assert streams[i]["footer"]["info"]["first_timestamp"][0] == "91725.21394789348"
+    assert streams[i]["footer"]["info"]["last_timestamp"][0] == "91735.21394789348"
+    assert streams[i]["footer"]["info"]["sample_count"][0] == "10"
+    first_clock_offset = streams[i]["footer"]["info"]["clock_offsets"][0]["offset"][0]
+    assert first_clock_offset["time"][0] == "91716.6915717245"
+    assert first_clock_offset["value"][0] == "-1.94335007108748e-05"
+    last_clock_offset = streams[i]["footer"]["info"]["clock_offsets"][0]["offset"][-1]
+    assert last_clock_offset["time"][0] == "91746.69274602149"
+    assert last_clock_offset["value"][0] == "-3.694550105137751e-05"
+
+    # Time-series data
+    s = np.array([[0], [1], [2], [3], [4], [5], [6], [7], [8], [9]], dtype=np.int32)
+    t = np.array(
+        [
+            91725.21394789,
+            91726.21394789,
+            91727.21394789,
+            91728.21394789,
+            91729.21394789,
+            91730.21394789,
+            91731.21394789,
+            91732.21394789,
+            91733.21394789,
+            91734.21394789,
+        ]
+    )
+
+    np.testing.assert_equal(streams[i]["time_series"], s)
+    if synchronize_clocks:
+        np.testing.assert_almost_equal(streams[i]["time_stamps"], t, decimal=4)
+    else:
+        np.testing.assert_almost_equal(streams[i]["time_stamps"], t, decimal=8)
+
+    # Dejittering should have no effect because timestamps have minimal jitter.
```

Comment:
I know this is super nitpicky, but I'd prefer comments to not end with a period. Can you remove them here (and elsewhere if present)?

Reply:
All good I think this was the only one here, but there's quite a few in my other PRs but I'll fixup those before we move on.